gems/aws-sdk-core/lib/aws-sdk-core/cbor/engines/cbor_engine.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require_relative '../tagged'
+require_relative '../encoder'
```

Comment:
This still feels like a weird structure to me.
The encoder and decoder are specific to this engine and its implementation, they feel like they should be a lower/same level and not in the top level Cbor module.